src/Snifter/IPPacket.cs

```diff
@@ -10,8 +10,8 @@ public class IPPacket
         public int Protocol { get; private set; }
         public IPAddress SourceAddress { get; private set; }
         public IPAddress DestAddress { get; private set; }
-        public short SourcePort { get; private set; }
-        public short DestPort { get; private set; }
+        internal UInt16 SourcePort { get; private set; }
```

Comment:
As in `AppOptions`, would prefer `ushort` instead of `UInt16`

Reply:
Okay, please take a look here too.